docker-compose.yml

```diff
@@ -1,60 +1,84 @@
 version: "3"
 services:
 
-  exodus:
+  amqp:
+    image: rabbitmq:3.7-alpine
+    networks:
+      backend:
+        aliases:
+            - amqp
+    ports:
+      - "5672:5672"
+
+  db:
+    environment:
+      POSTGRES_USER: exodus
+      POSTGRES_PASSWORD: exodus
+      POSTGRES_DB: exodus
+      PGDATA: /var/lib/postgresql/data/pgdata
+    image: postgres:11
+    restart: always
+    networks:
+      backend:
+        aliases:
+            - db
+    ports:
+      - "5432:5432"
+    volumes:
+      - db:/var/lib/postgresql/data/pgdata
+
+  exodus-front:
     build:
       context: ./
       dockerfile: ./docker/Dockerfile
-    container_name: exodus
-    image: exodus
-    volumes:
-      - ./exodus:/exodus/exodus:rw,cached
     depends_on:
       - minio
       - db
       - amqp
+      - exodus-worker
+    env_file: .env
+    networks:
+      backend:
+        aliases:
+            - exodus
     ports:
       - "80:8000"
+    user: ${uid?'echo uid=$(id -u) >> .env'}
```

Comment:
I don't know if this is supposed to work that way, but I had to run manually the command myself. It's not that big an issue, but if that's the case maybe indicate it in the documentation?

Reply:
The idea is that the exe should be run as the developer, so files won't be created or modified with a different UID that the developer's. `'echo uid=$(id -u) >> .env'` is supposed to be “documentation” to indicated how to add the UID to the .env file.

I didn't want to be too verbose, I do agree that there is a need for a more friendly message.